rcl/src/rcl/timer.c

```diff
@@ -33,7 +33,7 @@ typedef struct rcl_timer_impl_t
   rcl_clock_t * clock;
   // The associated context.
   rcl_context_t * context;
-  // A guard condition used to wake a wait set if using ROSTime, else zero initialized.
+  // A guard condition used to wake a wait set
```

Comment:
Instead of removing details, please add it, something like:

```c
// A guard condition used to wake the associated wait set, either when
// ROSTime causes the timer to expire or when the timer is reset.
```

Reply:
I change the comment of guard condition.